crates/epaint/src/image.rs

```diff
@@ -101,6 +101,21 @@ impl ColorImage {
         Self { size, pixels }
     }
 
+    /// Create a [`ColorImage`] from flat RGB data.
+    ///
+    /// This is what you want to use after having loaded an image file (and if
+    /// you are ignoring the alpha channel - considering it to always be 0xff)
+    ///
+    /// Panics if `size[0] * size[1] * 3 != rgba.len()`.
```

Comment:
Misprint or copy-paste error:
```suggestion
    /// Panics if `size[0] * size[1] * 3 != rgb.len()`.
```